Fix: Changes the navbar-logo for better visual appeal in both the dark and light themes.

ayush00git opened the pull request:
Fixes #376 
Changes i've made -
Edited the `static/img/navbar-logo.png`, now it's visually appealing in both the light and dark theme of the website.

Here's how it looks - 
In the dark mode -
<img width="1920" height="536" alt="image" src="https://github.com/user-attachments/assets/275823ac-010b-4dfb-9bcf-fd9b1ace1040" />

In the light mode - 
<img width="1920" height="536" alt="image" src="https://github.com/user-attachments/assets/5f30de48-4dd6-4e50-98b8-d6ce41c9884b" />

chaokunyang commented:
> > > o make it more dark for light m
> > 
> > 
> > maybe we still need to implement this by css
> 
> Okk means i should try to render two different images for dark and light modes using css properties? Should i go with this approach?

Yes, we should go with this approach

ayush00git commented:
Hey @chaokunyang,
Take a look now - 

<img width="1905" height="491" alt="image" src="https://github.com/user-attachments/assets/f2259354-9fcf-4728-a94a-aa9aed18921c" />

<img width="1905" height="491" alt="Screenshot From 2025-12-18 16-18-28" src="https://github.com/user-attachments/assets/0c790127-8905-4628-898d-bdd27a20a6c7" />

Changes i made - 

1. In `src/css/custom.css` I added CSS rules to make sure the correct logo is displayed based on the active theme.
2. In `docusaurus.config.ts`  I updated the `navbar.logo` configuration to specify both a light and dark version of the logo.
